server/mdm/android/service/service.go

```diff
@@ -910,9 +910,9 @@ func (svc *Service) AddFleetAgentToAndroidPolicy(ctx context.Context, enterprise
 ) error {
 	var errs []error
 	if packageName := os.Getenv("FLEET_DEV_ANDROID_AGENT_PACKAGE"); packageName != "" {
-		sha256Fingerprint := os.Getenv("FLEET_DEV_ANDROID_AGENT_SHA256")
+		sha256Fingerprint := os.Getenv("FLEET_DEV_ANDROID_AGENT_SIGNING_SHA256")
 		if sha256Fingerprint == "" {
-			return ctxerr.New(ctx, "FLEET_DEV_ANDROID_AGENT_SHA256 must be set when FLEET_DEV_ANDROID_AGENT_PACKAGE is set")
+			return ctxerr.New(ctx, "FLEET_DEV_ANDROID_AGENT_SIGNING_SHA256 must be set when FLEET_DEV_ANDROID_AGENT_PACKAGE is set")
 		}
 		for uuid, managedConfig := range hostConfigs {
 			policyName := fmt.Sprintf("%s/policies/%s", enterpriseName, uuid)
```

Comment:
In this file, in `BuildAndSendFleetAgentConfig` do we need to change...?

line 1192
```

			newCertsByHost[template.HostUUID] = append(newCertsByHost[template.HostUUID], fleet.HostCertificateTemplate{
				HostUUID:              template.HostUUID,
				CertificateTemplateID: template.CertificateTemplateID,
				FleetChallenge:        challenge,
				Status:                fleet.MDMDeliveryPending,
			})
```

`Status:                fleet.MDMDeliveryPending,` <-

Or are we going to insert `pending` and remap on reads?